ols/src/query_helpers/docs_summarizer.py

```diff
@@ -92,6 +89,9 @@ def summarize(
 
         embed_model = "local:BAAI/bge-base-en"
 
+        bare_llm = self.llm_loader(
+            self.provider, self.model, llm_params=self.llm_params
```

Comment:
this starts to raise an interesting question as to whether llm_params are more closely associated with the model you're invoking, or with the use case (in this case docs summarization).

nothing to do for now, but something to think about as we continue down the path of tweaking llm parameters

cc @asamal4 @williamcaban

Reply:
The params that are exposed are a combination of model type + functionality (e.g. embedding vs completion/query) + provider. 

When using the models for "embedding" (e.g., /embeddings or /api/embeddings)  there are parameters like padding and embedding dimensions that are configurable for the model but not all providers or loaders support configuring them.

When the model is used for "generate" or "chat" (e.g. /completion, /generate, /api/generate, /chat, /api/chat or /v1/chat/completions), the parameters are different (e.g. top_k, temperature, etc.) and the provider may also recognized additional parameters (e.g. num_thread, seed, num_gpu).

If want to use default settings and not pass any params, just define an empty array `{}` for the llm_params argument.